### PR TITLE
Fix Licenses

### PR DESCRIPTION
**Pull request by kgugala**
This PR fixes copyrights in the LICENSE file and some license headers in the CI scripts

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020-2021 The SymbiFlow Authors
```
Comment (mithro) [inline LICENSE:190]: You never update this file. Please revert this change.

Comment (kgugala) [reply LICENSE:190]: ok

Comment (kgugala) [reply LICENSE]: done

**Review by mithro — APPROVED** [review]
LGTM.
